src/main/resources/scss/abstracts/_overrides.scss

```diff
@@ -6,7 +6,7 @@
   display: flex !important;
   flex-direction: column;
   width: 100%;
-  max-width: max(950px, 65vw);
+  max-width: max(950px, 55vw);
```

Comment:
seems a bit odd to me on a large monitor (27")

a little far to the right

![image](https://user-images.githubusercontent.com/21194782/188468325-a717f4c7-6a37-49c2-8616-b8fccdc643c3.png)

Reply:
Undone that change, I think it would be worth looking into in the future though as on larger monitors the library can get difficult to read due to how long the line length is.